src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(result, number) {
+      return result + number;
+    },
+
+    subtract(result, number) {
+      return result - number;
+    },
+
+    multiply(result, number) {
+      return result * number;
+    },
+
+    divide(result, number) {
+      return result / number;
+    },
```

Comment:
It's a bit weird to pass a `result `inside an object that already has a `result `field. But I don't know any other options so that you can not fix it.